modules/scaleform.lua

```diff
@@ -0,0 +1,208 @@
+---@class Scaleform : OxClass
+---@field name string
+---@field scaleform number
+---@field draw boolean
+---@field renderTarget number
+---@field targetName string
+---@field handle number
+local Scaleform = lib.class('Scaleform')
+
+---@param name string
+---@return nil
+---@description Create a new scaleform class
+function Scaleform:constructor(name)
```

Comment:
We should let users provide everything with a table of arguments.
"Everything" being the name, fullscreen, and properties.